todo_app/app.py

```diff
@@ -1,22 +1,79 @@
 from flask import Flask, render_template, request
-from .data import session_items as sessionitems
 from todo_app.flask_config import Config
+import json
+import os
+import requests
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    items = sessionitems.get_items()
+    allCards = requests.get('https://api.trello.com/1/boards/602410634db0296b392dc99c/cards/open?key={}&token={}'.format(os.environ["TRELLO_KEY"], os.environ["TRELLO_TOKEN"] )).json()
+    items = []
+    for item in allCards:
+        if item['idList'] == "602410634db0296b392dc99d":
+            items.append(item['name'])
     return render_template('index.html', items=items)
 
 @app.route('/add_item', methods=['POST'])
 def add():
     todoTitle = request.form.get('title')
-    sessionitems.add_item(todoTitle)
-    items = sessionitems.get_items()
+    url = "https://api.trello.com/1/cards"
+
+    query = {
+    'key': os.environ["TRELLO_KEY"],
+    'token': os.environ["TRELLO_TOKEN"], 
+    'idList': '602410634db0296b392dc99d',
+    'name': todoTitle
+    }
+
+    response = requests.request(
+    "POST",
+    url,
+    params=query
+    )
+
+    allCards = requests.get('https://api.trello.com/1/boards/602410634db0296b392dc99c/cards/open?key={}&token={}'.format(os.environ["TRELLO_KEY"], os.environ["TRELLO_TOKEN"] )).json()
+    items = []
+    for item in allCards:
+        if item['idList'] == "602410634db0296b392dc99d":
+            items.append(item['name'])
+    return render_template('index.html', items=items)
```

Comment:
Redirecting to the index (`return redirect('/')`) works better with the browser, otherwise you'll find if you add a new card and then refresh the page you'll end up adding another one!